source/commands.txt

```diff
@@ -149,8 +149,11 @@ Run these commands from the Command Palette to create and run
        Your playground runs against the deployment specified in your
        active connection.
        
-       |vsce| outputs the results of your playground in the
-       :guilabel:`Output` view in VS Code.
+       |vsce| splits your playground window to display the results in 
+       a pane titled :guilabel:`Playground Results.json` to the right 
+       of your playground. If you disabled split-view for your VS Code 
+       Editor window, |vsce| displays the playground results in a new 
+       tab to the right of your playground.
 
```

Comment:
Also when you manually moved the playground results somewhere, vscode displays the playground results in the tab to where it was moved.